packages/renderer/src/lib/image/RunImage.svelte

```
@@ -597,8 +579,8 @@ function onContainerPortMappingInput(event: Event, index: number): void {
 }
 
 function onHostContainerPortMappingInput(event: Event, index: number): void {
-  onPortInput(event, hostContainerPortMappings[index].hostPort, () => {
-    hostContainerPortMappings = hostContainerPortMappings;
+  onPortInput(event, options.basic.hostContainerPortMappings[index].hostPort, () => {
+    options.basic.hostContainerPortMappings = options.basic.hostContainerPortMappings;
```

Biome (2.5.1)

[error] hostContainerPortMappings is assigned to itself.

(lint/correctness/noSelfAssign)